integration/simulation/network/obscuro_node_utils.go

```diff
@@ -150,25 +150,25 @@ func startStandaloneObscuroNodes(params *params.SimParams, stats *stats.Stats, g
 	walletClients := make(map[string]rpcclientlib.Client)
 	var i int
 	for _, w := range params.Wallets.SimObsWallets {
-		cli, err := rpcclientlib.NewClient(nodeRPCAddresses[i%len(nodeRPCAddresses)])
+		cli, err := rpcclientlib.NewViewingKeyNetworkClient(nodeRPCAddresses[i%len(nodeRPCAddresses)])
```

Comment:
@BedrockSquirrel Not your fault, but any chance you could rename `cli` to `client`? `cli` is reserved for CLIs IMO

Reply:
Probably was my fault originally tbf haha